pyflakes/messages.py

```diff
@@ -233,6 +233,13 @@ class TooManyExpressionsInStarredAssignment(Message):
     message = 'too many expressions in star-unpacking assignment'
 
 
+class IfTuple(Message):
+    """
+    Conditional test is a tuple, which are always True.
+    """
+    message = 'if condition is always true, perhaps remove parentheses?'
```

Comment:
the messaging here probably needs to talk about it being a tuple -- removing parentheses doesn't fix it -- maybe something like '`if tuple literal` is always true, perhaps remove an accidental comma?'

Reply:
D'oh, this is me copying too much from the (otherwise fairly similar) `AssertTuple` case. That wording sounds good though :+1: